standards/B3.md

```diff
@@ -0,0 +1,228 @@
+# ERC948 Interface for Pull Requests - B3's Financial Transaction Business Process Protocol (FTBPP)
+### Summary
+A standard interface to be implemented by smart wallets that wish to allow pre-authorized pull requests.
+
+### Abstract
+This standard defines an interface to pre-authorize payments in order to allow parties to fulfill the payment at a later date via a pull request. The approval/pull request mechanism is purposefully generalized so that it can be used for both recurring pull requests (subscriptions) and one-off pull requests. This standard allows for payment to occur in Ether, ERC20, ERC223 tokens. This should be integrated into future versions of smart wallets, allowing for a more seamless process.
+
+### Our Thoughts
+The ability to support all types of pull requests, not only subscriptions, is of utmost importance. Subscription is just one subset of an abundance of use cases for pull requests. Therefore, because we cannot now predict all of the unique uses for pull requests, it is important to create a standard that is general, yet secure enough to allow creative developers to build upon without the need for creating a new standard. 
```

Comment:
This is well thought and described. I think in general most of the proposal are using the same idea (in general subscriptions payment is solving the problem of transition from push to pull) but without explicitly naming that.